### PR TITLE
[EOC-98] History list on right two columns change name to done leave green tick on done items

### PR DESCRIPTION
**Pull request by alemikolo**
I changed item's styles a little bit to adjust its look according to Łukasz suggestions.
chrome-capture
chrome-capture (1)

### CODE REVIEW

File: client/src/modules/list/components/ItemsListItem.jsx [hunk 1]
```diff
@@ -1,54 +1,72 @@
...
-      <span className="items-list__data">
-        <span>{name}</span>
-        <span className="items-list__author">{`Ordered by: ${authorName}`}</span>
-      </span>
-      {!archived && (
-        <VotingBox
-          voteForItem={voteForItem}
-          votesCount={votesCount}
-          whetherUserVoted={whetherUserVoted}
+class ItemsListItem extends PureComponent {
```
Comment (marekrozmus) [inline client/src/modules/list/components/ItemsListItem.jsx]: change just to `ListItem`

File: client/src/app/styles/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@mixin medium-screens(){
+  @media only screen and (min-width: 800px) {
+    @content;
+  }
+}
```
Comment (adamklepacz) [inline client/src/app/styles/_mixins.scss]: More flexible solution will be to define breakpoints as const in some sass file maybe _variables.scss 

```
 $small: 0px,
 $medium: 640px,
 $large: 1024px,
 $xlarge: 1200px,
 $xxlarge: 1440px,
```

and then write mixin
```
@mixin breakpoint($width) { 
   @media only screen and (min-width: $width) {
      @content;
    }
}
```

and then we will simply use 
```
@include breakpoint($medium) {
 ...some styles here
}
```

File: client/src/modules/list/components/ItemsContainer.scss [hunk 3]
```diff
@@ -3,6 +3,11 @@
   background-color: $container-bg;
   margin-bottom: 20px;
   text-align: center;
+  width: 100%;
+
+  @include medium-screens {
+    width: 48%;
+  }
```
Comment (adamklepacz) [inline client/src/modules/list/components/ItemsContainer.scss]: and here we will have a more intuitive piece of code!
```
@include breakpoint($medium) {
   width: 48%;
}
```

File: client/src/app/styles/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// Screen sizes
+$tiny: 360px;
+$small: 640px;
+$medium: 800px;
+$normal: 1024px;
+$large: 1200px;
+$xlarge: 1440px;
```
Comment (adamklepacz) [inline client/src/app/styles/_variables.scss]: start with $small and add extra $xxlarge variable.
convention of this variables is to keep as cloaths sizes so: s,m,l,xl, xxl etc

**Comment by adamklepacz** [comment]
Zrzut ekranu 2019-03-15 o 13 38 25
Add some white space over arrow.

**Pull request merged**